src/components/CreateNimi/CreateNimi.tsx

```diff
@@ -340,23 +345,11 @@ export function CreateNimi({ ensAddress, ensName, provider }: CreateNimiProps) {
                 if (newArray) setValue('addresses', newArray);
               }
 
-              // const arrayOfWidgetsItemsToBeRemoved = formWidgetList.filter((item) => !nimiWidgetList.includes(item));
-              // if (arrayOfWidgetsItemsToBeRemoved.length > 0) {
-              //   const formData = getValues('widgets');
-              //   const newArray = formData.filter((item) => !arrayOfWidgetsItemsToBeRemoved.includes(item.type));
-
               setValue(
                 'widgets',
-                widgets.map((widget) => {
-                  if (widget === NimiWidgetType.POAP) {
-                    return {
-                      type: NimiWidgetType.POAP,
-                      address: ensAddress,
-                    };
-                  }
-
-                  return widget;
-                })
+                widgets.map((widget) => ({
+                  type: widget,
+                }))
```

Comment:
I think this should return the widget type and options if any.

Reply:
Will the options be defined by default? If so then yeah, it needs to be updated.